### ISSUE TITLE
MSFT-242[Registration] JAWS : Banner region is announced differently in Windows - Chrome

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#16]
**Steps to Reproduce**
1. Launch JAWS 
2. Launch "https://accounts.topcoder.com/member/registration" in Windows / Chrome
3. Navigate using R (for Regions)

**Expected Result**
It should be announced as "Banner"

**Actual Result**
Note that "Banner region" is announced as "vm dot register form form"

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome
**WCAG Category**: 3. Understandable
**WCAG Level**: AAA
**WCAG Success Criterion**: 3.1.3 Unusual Words

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FJAWS%20Banner%20region%20announced%20differently%20in%20Chrome
